README.md

```diff
@@ -4,6 +4,8 @@ Assuming your local environment is setup correctly with Golang standards, you ca
 
 `cd to this repo`
 
+`export PORT=8080`
```

Comment:
So Heorku uses environment variables for configurations. And it also has a deploy local command, that will populate certain environment fields from a json file. I'll look into it but once i've set that up we will no longer need to do this manually.